cluster/src/main/java/org/apache/iotdb/cluster/rpc/raft/impl/RaftNodeAsClientManager.java

```diff
@@ -226,17 +285,16 @@ public void syncHandleRequest(BasicRequest request, PeerId leader,
         qpTask.run(null);
         throw new RaftConnectionException(e);
       } finally {
-        releaseClient();
+        releaseClient(this);
```

Comment:
why not RaftNodeAsClient.this?

Reply:
It's the same, but pointing it out clearly is better.